main.go

```diff
@@ -113,7 +112,6 @@ var (
 )
 
 func main() {
-	rand.NewSource(time.Now().UnixNano())
```

Comment:
Why is this changing? I never noticed this line before, what was this used for?

Reply:
Good point, I myself have not noticed this. I dont think this line has any affect, that's why it was removed by @furkatgofurov7

Reply:
Yep. The only place we are using Random is in subnets.go to shuffle the AZs. But `rand.Shuffel` is already using a default seed. So it will provide pseudo random results every time its running. Removing this should be safe.